### PR TITLE
Minor JSDoc improvements

### PR DESCRIPTION
**Pull request by koloml**
Minor JSDoc improvements for several classes.
Also contains editorconfig changes about comments wrapping and maximal line size.

Note: Konstantin is actually me, but on the different machine. I forgot that I have different default git configuration for pushing.

### CODE REVIEW

**Comment by koloml** [comment]
@Kolonsky also, can you check editorconfig entries for maximal line size and wrapping long comment blocks for VSCode?

File: .editorconfig [hunk 1]
```diff
@@ -9,7 +9,9 @@ end_of_line = lf
 charset = utf-8
 trim_trailing_whitespace = true
 insert_final_newline = true
+max_line_length = 120
```
[the remark this answers is not in the document]
Comment (koloml) [reply .editorconfig:12]: I read it. But if this config entry is not set then IntelliJ IDEA will wrap comments incorrectly. For example, somone can set size to 90 chars and comments wrapping will work for this amount of chars.

[the remark this answers is not in the document]
Comment (koloml) [reply .editorconfig:12]: Is it not working in VCS at all?

[the remark this answers is not in the document]
Comment (Kolonsky) [reply .editorconfig:12]: No, it's not.
But, we can use ESLint to follow JavaScript standards. It is supported in both VS Code and Intellij IDEA.

**Pull request merged**